.github/workflows/_build-wheel-release-upload.yml

```diff
@@ -13,8 +13,8 @@ on:
         default: false
         required: false
         type: boolean
-      github_admin_username:
-        description: 'GitHub admin authorized for GitHub/PyPI release'
+      maintainer_github_username:
```

Comment:
Instead of `github_maintainer_username`, I used `maintainer_github_username` because we the concept of "maintainer" doesn't fit within `github` but rather within our `skpkg` framework.